src/frontend/ui/capture/index.ts

```diff
@@ -89,6 +91,17 @@ export interface CaptureVideoOptions extends CaptureOptions {
 }
 
 export class Capture extends KojiBridge {
+  /**
```

Comment:
I'm wondering here if we can be more explicit by doing something like this:

```
/**
   * Prompts the user to select a color, either from a swatch or by entering a color code.
   * Supports HEX, RGB, or HSL by default, and RBGA or HSLA, if transparency is enabled.
   *
   * @param {CaptureColorOptions} options The options to pass
   * @returns {(string|CaptureValue)} Returns the string value, or an extended object if the verbose option is true.
   * @example
   * ```javascript
   * const color = await Koji.UI.Capture.color();
   * ```
   */
```

And then above would be a typedef:

```
/**
 * The extended return value from a ui capture source
 * @typedef CaptureValue
 * @param {string} event The event
 * @param {string} result The return value
 * @param {string} status Whether the user succeeded
 * @param {string} type The type of the event
 */
export interface CaptureValue {
  event: string;
  result: string;
  status: string;
  type: string;
}
```

Reply:
Yeah, I want to ask about the best way to document interfaces, too. It wasn't working as expected in my tests, but I might have missed something. I don't know that @typedef is supported (at least, it's not listed in the typedoc docs).